packages/icons/src/types.ts

```diff
@@ -1,3 +1,50 @@
 import type { SVGProps } from "react";
 
-export type SvgComponentProps = SVGProps<SVGSVGElement>;
+export type SvgComponentProps = SVGProps<SVGSVGElement> & IconColorSet;
+
+export interface IconColorSet {
+  /**
+   * Main color of the icon
+   */
+  color?: IconColors;
+  /**
+   * Alternative color of the icon
+   */
+  colorAlt?: IconColors;
+  /**
+   * Accent color of the icon
+   */
+  colorAcc?: IconColors;
```

Comment:
Should we unify these with the names on the storybook UI?
![Screenshot 2024-05-10 at 16 39 38](https://github.com/giosg/design-system/assets/64484706/225cc6c1-5b27-433a-9581-44a91e86e97d)

Reply:
changed to desired